src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+/* #region card */
+$cardWidth: 200px;
+$secondaryColor: #616070;
+$buttonColor: #00acdc;
+$buttonTextColor: #fff;
+$mainColor : #060b35;
+
+.card {
+  box-sizing: border-box;
+  width: $cardWidth;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  margin-bottom: 50px;
+  transition: 0.5s;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__descreption {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $mainColor;
+    margin-bottom: 4px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $mainColor;
+    margin-bottom: 4px;
+  }
+
+  &__id {
+    color: $secondaryColor;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-text {
```

Comment:
`&__rate-text` is an element. Not a modifier. You should write this element inside parent class.